### PR TITLE
Add API for FE internship

### PR DESCRIPTION
**Pull request by workshur**
Test Assignment for FE internship 2022

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -1,35 +1,58 @@
...
-    6. Which framework would you choose for a new project? Explain why.
-#### [Code exercise](https://stackblitz.com/edit/js-llr1ac-async-task-chvwqk)
-
-#### Block 3: TypeScript
-    1. Is it possible to check types in runtime? If so, how?
-    2. What is the difference between private and protected fields?
-    3. How do interfaces help in development?
-    4. How would you pass arguments into a Class?
-#### [Code exercise](https://stackblitz.com/edit/typescript-2xfiqk)
+The repo contains the server part of a simple spend-tracking application written using ME*N stack.
```
Comment (majeX) [inline frontend/README.md:4]: `spend-tracking` => `spending-tracking` or `expense-tracking`.

File: frontend/README.md [hunk 2]
```diff
@@ -1,35 +1,58 @@
...
+- MongoDB - document database
+- Express(.js) - Node.js web framework
+- Any client-side JavaScript framework - React, Angular, Vue, Vanilla JS
+- Node(.js) - the premier JavaScript web server
+
+Express and Node make up the middle (application) tier. Express.js is a server-side web framework, and Node.js the popular and powerful JavaScript server platform.
+
+You need to write the client side of spend-tracking application using any JS framework / UI kit / libs you like.
+
+Application will need to display, add new, update, remove expenses and track statistics by month and category.
```
Comment (majeX) [inline frontend/README.md:16]: `category` => `spending category` for clarity.

File: frontend/README.md [hunk 3]
```diff
@@ -1,35 +1,58 @@
...
+
+## Getting Started
+
+Clone the repo
+```
+git clone git@github.com:provectus/internship.git
+cd /internship/frontend/exercise
+```
+
+Pre-fill the database with the generated data
```
Comment (majeX) [inline frontend/README.md:26]: `Pre-fill` => `Seed`.

File: frontend/README.md [hunk 4]
```diff
@@ -1,35 +1,58 @@
...
+```
+http://localhost:5000/api-docs
+```
+![Api](./swagger.png)
+## Acceptance criteria
+
+### Difficulty Level : Easy
+
+- Minimalistic UI implemented to display / create / update / delete expences
+- UI contains chart to display statistics by month and category
```
Comment (majeX) [inline frontend/README.md:47]: UI displays a chart that shows spending stats/statistics by month and category

File: frontend/README.md [hunk 5]
```diff
@@ -1,35 +1,58 @@
...
+- Added comprehensive documentation to launch the application
```
Comment (majeX) [inline frontend/README.md:48]: Just remove `added`.

File: frontend/README.md [hunk 6]
```diff
@@ -1,35 +1,58 @@
...
+
+### Difficulty Level : Medium
+
+- The app is written using TypeScript
```
Comment (majeX) [inline frontend/README.md:52]: `written using TypeScript` => `written in TypeScript`

Comment (majeX) [reply frontend/README.md:52]: Or just `use TypeScript` if you like to keep it short :)

File: frontend/README.md [hunk 7]
```diff
@@ -1,35 +1,58 @@
...
+- The forms contain validations
```
Comment (majeX) [inline frontend/README.md:53]: `The forms contain validations` => `Form input is validated`

File: frontend/README.md [hunk 8]
```diff
@@ -1,35 +1,58 @@
...
+
+### Difficulty Level : High
+
+- The client application is wrapped in a Docker container
+- The compose file updated, added section to run the whole application with one command
```
Comment (majeX) [inline frontend/README.md:58]: `The docker-compose file is updated and can be used to run the whole app with one command`

File: frontend/README.md [hunk 9]
```diff
@@ -1,35 +1,43 @@
...
-#### Block 3: TypeScript
-    1. Is it possible to check types in runtime? If so, how?
-    2. What is the difference between private and protected fields?
-    3. How do interfaces help in development?
-    4. How would you pass arguments into a Class?
-#### [Code exercise](https://stackblitz.com/edit/typescript-2xfiqk)
-
+[![forthebadge](https://forthebadge.com/images/badges/built-with-love.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/made-with-javascript.svg)](https://forthebadge.com)
+### Test Assignment
+The repository contains the backend part for a simple expense-tracking application written using ME\*N stack.
```
Comment (aMoRRoMa) [inline frontend/README.md:4]: It looks like `\` is superfluous
```suggestion
The repository contains the backend part for a simple expense-tracking application written using ME*N stack.
```

**Review by aMoRRoMa — APPROVED** [review]
Looks pretty cool!
Did you write it yourself or use some "scaffold"?

**Pull request merged**